diff --git a/nvim/after/plugin/colors.lua b/nvim/after/plugin/colors.lua
index 5479110..8762870 100644
--- a/nvim/after/plugin/colors.lua
+++ b/nvim/after/plugin/colors.lua
@@ -1,6 +1,8 @@
 -- require('onenord').setup({ theme = "light"})
 
-vim.cmd[[colorscheme nord]]
+-- vim.cmd[[colorscheme nord]]
+vim.cmd('set background=light')
+vim.cmd[[colorscheme zenbones]]
 
 -- -- Nord config
 -- vim.g.nord_contrast = true
diff --git a/nvim/after/plugin/lualine.lua b/nvim/after/plugin/lualine.lua
index 52d1eae..5f8c286 100644
--- a/nvim/after/plugin/lualine.lua
+++ b/nvim/after/plugin/lualine.lua
@@ -1,7 +1,7 @@
 require('lualine').setup({
   options = {
     icons_enabled = false,
-    theme = 'nord',
+    theme = 'auto',
     component_separators = '|',
     section_separators = '',
   },
diff --git a/tig/tigrc b/tig/tigrc
index 1c4ef87..b1f77ac 100644
--- a/tig/tigrc
+++ b/tig/tigrc
@@ -2,6 +2,10 @@
 set git-colors = no
 color cursor white 0 bold
 
+# Uncomment these for Zenbones Light
+set git-colors = no
+color cursor black white
+
 set wrap-lines = yes
 set main-view-author = full,width=30
 
diff --git a/tmux/tmux.conf b/tmux/tmux.conf
index a208723..b949be8 100644
--- a/tmux/tmux.conf
+++ b/tmux/tmux.conf
@@ -2,8 +2,8 @@
 #set -g default-terminal "screen-256color"
 
 # Uncomment these for Nord
-set -g default-terminal "xterm-256color"
-set-option -ga terminal-overrides ",xterm-256color:Tc"
+# set -g default-terminal "xterm-256color"
+# set-option -ga terminal-overrides ",xterm-256color:Tc"
 
 set -g history-limit 20000
 # use VI
@@ -124,5 +124,18 @@ set -g @plugin 'tmux-plugins/tmux-sensible'
 set -g @plugin "arcticicestudio/nord-tmux"
 set -g @plugin 'tmux-plugins/tmux-resurrect'
 
+# Uncomment this for Zenbones Light
+set -g status-left ' #[fg=#88507D,bold]#{s/root//:client_key_table} '
+set -g status-right '#[fg=#88507D,bold] [#S]#[fg=#88507D,bold] [%d/%m] #[fg=#88507D,bold][%I:%M%p] '
+set -g status-style fg='#88507D',bg='#2C363C'
+set -g window-status-current-style fg='#88507D',bg='#2C363C',bold
+set -g pane-border-style fg='#88507D'
+set -g pane-active-border-style fg='#88507D'
+set -g message-style fg='#F0EDEC',bg='#CBD9E3'
+set -g display-panes-active-colour '#88507D'
+set -g display-panes-colour '#88507D'
+set -g clock-mode-colour '#88507D'
+set -g mode-style fg='#F0EDEC',bg='#CBD9E3'
+
 # Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
 run '~/.tmux/plugins/tpm/tpm'
diff --git a/zenbones_light.patch b/zenbones_light.patch
index 7a1bad5..e69de29 100644
--- a/zenbones_light.patch
+++ b/zenbones_light.patch
@@ -1,61 +0,0 @@
-diff --git a/nvim/after/plugin/colors.lua b/nvim/after/plugin/colors.lua
-index 5479110..8762870 100644
---- a/nvim/after/plugin/colors.lua
-+++ b/nvim/after/plugin/colors.lua
-@@ -1,6 +1,8 @@
- -- require('onenord').setup({ theme = "light"})
- 
--vim.cmd[[colorscheme nord]]
-+-- vim.cmd[[colorscheme nord]]
-+vim.cmd('set background=light')
-+vim.cmd[[colorscheme zenbones]]
- 
- -- -- Nord config
- -- vim.g.nord_contrast = true
-diff --git a/nvim/after/plugin/lualine.lua b/nvim/after/plugin/lualine.lua
-index 52d1eae..5f8c286 100644
---- a/nvim/after/plugin/lualine.lua
-+++ b/nvim/after/plugin/lualine.lua
-@@ -1,7 +1,7 @@
- require('lualine').setup({
-   options = {
-     icons_enabled = false,
--    theme = 'nord',
-+    theme = 'auto',
-     component_separators = '|',
-     section_separators = '',
-   },
-diff --git a/tmux/tmux.conf b/tmux/tmux.conf
-index a208723..b949be8 100644
---- a/tmux/tmux.conf
-+++ b/tmux/tmux.conf
-@@ -2,8 +2,8 @@
- #set -g default-terminal "screen-256color"
- 
- # Uncomment these for Nord
--set -g default-terminal "xterm-256color"
--set-option -ga terminal-overrides ",xterm-256color:Tc"
-+# set -g default-terminal "xterm-256color"
-+# set-option -ga terminal-overrides ",xterm-256color:Tc"
- 
- set -g history-limit 20000
- # use VI
-@@ -124,5 +124,18 @@ set -g @plugin 'tmux-plugins/tmux-sensible'
- set -g @plugin "arcticicestudio/nord-tmux"
- set -g @plugin 'tmux-plugins/tmux-resurrect'
- 
-+# Uncomment this for Zenbones Light
-+set -g status-left ' #[fg=#88507D,bold]#{s/root//:client_key_table} '
-+set -g status-right '#[fg=#88507D,bold] [#S]#[fg=#88507D,bold] [%d/%m] #[fg=#88507D,bold][%I:%M%p] '
-+set -g status-style fg='#88507D',bg='#2C363C'
-+set -g window-status-current-style fg='#88507D',bg='#2C363C',bold
-+set -g pane-border-style fg='#88507D'
-+set -g pane-active-border-style fg='#88507D'
-+set -g message-style fg='#F0EDEC',bg='#CBD9E3'
-+set -g display-panes-active-colour '#88507D'
-+set -g display-panes-colour '#88507D'
-+set -g clock-mode-colour '#88507D'
-+set -g mode-style fg='#F0EDEC',bg='#CBD9E3'
-+
- # Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
- run '~/.tmux/plugins/tpm/tpm'
